cmd/http/main.go

```diff
@@ -148,11 +148,11 @@ func (handler *KeyValueHandler) getKV(w http.ResponseWriter, r *http.Request) (s
 				return "", err
 			}
 
+			revs := handler.indexTree.RangeSince(ctx, []byte(key[0]), nil, int64(fromRev))
+
 			{
-				_, span := otel.Tracer(config.TraceName).Start(ctx, "rangesince kv", trace.WithSpanKind(trace.SpanKindClient))
+				_, span := otel.Tracer(config.TraceName).Start(ctx, "get kv", trace.WithSpanKind(trace.SpanKindClient))
```

Comment:
If you add traces within the RangeSince method, you don't need to start another tracer outside, just like you did in the put, get, and tombstone.

Reply:
it serves different purpose. The one span from line 154 is for "remote" reading from host nodes. we do need to trace it besides the rangesince which is local index op.

Reply:
Got it. Do you mean the tracer "get kv" is to trace the latency of "getValuesByRevs" in line 156?

Reply:
yes.

Reply:
It might be better to change the tracer name to the method "getValuesByRevs" to avoid confusing.